src/App.vue

```diff
@@ -40,8 +44,32 @@ export default {
 
 <style>
 body {
-  padding-top: 70px;
+  padding-top: 45px;
+}
+.navbar-custom {
+  height: 45px;
+  background: rgb(46,143,177); /* Old browsers */
+  background: -moz-linear-gradient(top,  rgba(46,143,177,1) 0%, rgba(33,134,169,1) 53%, rgba(13,119,153,1) 100%); /* FF3.6+ */
```

Comment:
> rgba(46,143,177,1)

I don't think this is a good idea. IMO it could be abstracted with CSS pre-processors like sass (one bootstrap uses, afaik). And the same applies to every other customization.